src/hotspot/share/gc/shared/cardTable.hpp

```diff
@@ -195,7 +199,7 @@ class CardTable: public CHeapObj<mtGC> {
 
   static constexpr CardValue clean_card_val()          { return clean_card; }
   static constexpr CardValue dirty_card_val()          { return dirty_card; }
-  static constexpr intptr_t clean_card_row_val()   { return clean_card_row; }
+  static constexpr CardWord clean_card_word_val()   { return clean_card_word; }
```

Comment:
Body is weirdly misaligned with the others.  I'm personally not a fan of that sort of alignment, but a mix
of alignments is even worse.